docs/guides/upgrade.rst

```diff
@@ -41,17 +41,17 @@ Upgrade your node
 
 .. code-block:: bash
 
-    docker run --rm -ti \
-        -v $(pwd)/keys:/opt/pyaleph/keys \
-        -v $(pwd)/node-secret.key:/opt/pyaleph/node-secret.key:ro \
-        -v $(pwd)/config.yml:/opt/pyaleph/config.yml:ro \
-        alephim/pyaleph-node:beta \
-        python3 /opt/pyaleph/migrations/config_updater.py \
+    docker-compose [-f <docker-compose-file>] down
+
+    docker-compose [-f <docker-compose-file>] \
+            run pyaleph /opt/pyaleph/deployment/migrations/config_updater.py \
```

Comment:
Incorrect path, the correct path is `/opt/pyaleph/migrations/config_updater.py`.

Reply:
The file was missing from the container.
I updated the commit to add it.

Reply:
It's not missing, it's just not copied in the same place.